Mediapipe.Net/Calculators/BlazePoseCpuCalculator.cs

```diff
@@ -10,6 +10,6 @@ namespace Mediapipe.Net.Calculators
     public class BlazePoseCpuCalculator : CpuCalculator<NormalizedLandmarkListPacket, NormalizedLandmarkList>
     {
         protected override string GraphPath => "mediapipe/graphs/pose_tracking/pose_tracking_cpu.pbtxt";
-        protected override string? SecondaryOutputStream => "pose_landmarks";
+        protected override string SecondaryOutputStream => "pose_landmarks";
```

Comment:
The `SecondaryOutputStream` property is nullable because you don't necessarily want one. Maybe you just want a `CpuCalculator` that outputs an image frame and that's it.

Reply:
but all these graphs do have it. the base class is nullable, but if you don't want the secondary output you can just not bind anything to `OnResult`

Reply:
That's not what I mean: I mean that if you want to have a calculator that doesn't compute any kind of secondary output, meaning it doesn't have any secondary output stream to listen to, then you can just make that null.

Reply:
and you can do that, on the base class.
https://github.com/vignetteapp/MediaPipe.NET/blob/93f3e05d176ac87de060598aef05a878eed59666/Mediapipe.Net/Calculators/CpuCalculator.cs#L28

https://github.com/vignetteapp/MediaPipe.NET/blob/93f3e05d176ac87de060598aef05a878eed59666/Mediapipe.Net/Calculators/GpuCalculator.cs#L31

Reply:
Yes.
So when overriding it doesn't matter that it's nullable? That's weird because I thought I had errors for this.

Reply:
i still think it's unnecessary, but fixed it